src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -37,6 +40,23 @@
         );
     }
 
+    public static Stream<Arguments> badArgumentsProvider() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(), 400),
+                Arguments.of(Arrays.asList(""), 400)
+        );
+    }
+
+    public static Stream<Arguments> correctArgumentsProvider() {
+        return Stream.of(
+                Arguments.of("amer", Arrays.asList("United States of America"), 200),
```

Comment:
If all arguments contain the same value, why do you need to pass it in the argumentProvider?
I'm talking about 200, you pass it in all cases.
The same for 400 in the badArgumentsProvider